### PR TITLE
Feature/phase 6

### PR DESCRIPTION
**Pull request by YuhsinLiao1995**
## Description
added Phase 6 tab and integrated the query with polling for real-time data

## Checklist
- [v] Ran Linting
- [v] Targeted PR against correct branch.
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [v] Wrote unit tests.
- [ ] Added an entry to the `CHANGELOG.md` file.
- [v] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: public/static/locales/en/desmos_primer.json [hunk 1]
```diff
@@ -28,5 +29,10 @@
   "dsmGained": "DSM Gained",
   "isReferred": "Is Referred",
   "referredCount": "Referred Count",
-  "totalDsmGiven": "Total DSM Given"
+  "totalDsmGiven": "Total DSM Given",
+  "createdRelationship": "Created RelationShip",
```
Comment (ryuash) [inline public/static/locales/en/desmos_primer.json:33]: relationship not RelationShip

File: src/screens/desmos_primer/components/phase_six/hooks.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+
+export const usePhaseFiveHook = () => {
```
Comment (ryuash) [inline src/screens/desmos_primer/components/phase_six/hooks.tsx:3]: usePhaseSixHook

File: src/screens/desmos_primer/components/phase_six/hooks.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+  return {
+    page,
+    setPage,
+    rowsPerPage,
+    setRowsPerPage,
+    handleChangePage,
+    handleChangeRowsPerPage,
+  };
+};
```
Comment (ryuash) [inline src/screens/desmos_primer/components/phase_six/hooks.tsx:24]: Actually I just took a look at the code. Can you show me where this is being imported and used??

File: src/screens/desmos_primer/components/phase_six/index.tsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  TablePagination,
+  TableRow,
+  TableSortLabel,
+} from '@material-ui/core';
+import { useTranslation } from 'i18n';
+import { Row } from '@components';
+import { IProps } from './types';
+import { useTableHook } from '../../hooks';
+
+const PhaseFour = (props:IProps) => {
```
Comment (ryuash) [inline src/screens/desmos_primer/components/phase_six/index.tsx:18]: PhaseSix

File: src/screens/users/hooks.tsx [hunk 5]
```diff
@@ -20,6 +20,8 @@ import {
 import { joinData } from './utils';
 import { State } from './types';
 
+// const er = DesmosPrimerAll.
```
Comment (ryuash) [inline src/screens/users/hooks.tsx:23]: remove unused code

**Review by ryuash — CHANGES_REQUESTED** [review]
Looks great!!! Can you just fix the typos and remove any unused code? :D

File: src/screens/desmos_primer/components/phase_six/hooks.tsx (hunk 3 above)
Comment (YuhsinLiao1995) [reply src/screens/desmos_primer/components/phase_six/hooks.tsx:24]: i found it is never used, so i deleted phase_six/hooks. and phase_five/hooks.

**Pull request merged**